dsn.go

```diff
@@ -139,11 +139,17 @@ func (c *Config) ocspMode() string {
 
 // DSN constructs a DSN for Snowflake db.
 func DSN(cfg *Config) (dsn string, err error) {
-	if cfg.Region == "us-west-2" {
+	if strings.ToLower(cfg.Region) == "us-west-2" {
 		cfg.Region = ""
+		logger.Info("Ignoring Region configuration in DSN as us-west-2 is the default one.")
```

Comment:
I don't think it is worth being logged at info, debug should suffice.

Reply:
the goal here is for the user is to see it and correct it to get rid of the log message. 
initially i thought debug too, but then it would be probably hidden and never corrected.

Reply:
And I agree with the cases below when customer includes region in account name, but here I'd say it is fine to give specific region as a separate parameter. But I'm not pushing you, I'll let you decide the final option.

Reply:
removed this logline for when cfg.Region is us-west-2